src/bin/commands/startup.rs

```diff
@@ -0,0 +1,22 @@
+use crate::config::default_ctants::{HOST_ADDRESS, PORT};
```

Comment:
As mentioned we can have those constants currently in the main binary file and remove the module `config`.

Reply:
Done.